docs/sources/clients/promtail/installation.md

```diff
@@ -3,8 +3,8 @@ title: Installation
 ---
 # Install Promtail
 
-Promtail is distributed as a [binary](#binary), [Docker container](#docker), and
-[Helm chart](#helm).
+Promtail is distributed as a binary, in a Docker container,
```

Comment:
nit: can we still keep the `#binary`, `#docker` and `#helm` links?

Reply:
Agreed, I think we should

Reply:
I hear you, but if you build and look at the docs, those links are to subsections that are very short, and all appear easily on even the smallest size browser window just below the links. If a reader actually clicked on the links, they'd see exactly the same thing (the page wouldn't even move) since the web page is short. 

I removed the links, as they don't help the reader find anything, and may contribute to user confusion, since "nothing" will happen if the reader clicks on the links.

Hearing my reply, do you still want me to add those links back in @kavirajk and @dannykopping ?

Reply:
@KMiller-Grafana I see what you mean. It was just a suggestion(not blocking the PR to merge). But I still think it's nice to keep the links, It may look short now, but as docs may grow, this will definitely useful.

Reply:
You've convinced me, thanks @KMiller-Grafana

Reply:
@kavirajk I think because those headings are so closely co-located, it should be fine even if the doc grows